beacon-chain/blockchain/head.go

```diff
@@ -234,7 +231,10 @@ func (s *Service) setHeadInitialSync(root [32]byte, block interfaces.SignedBeaco
 // This returns the head slot.
 // This is a lock free version.
 func (s *Service) headSlot() types.Slot {
-	return s.head.slot
+	if s.head == nil || s.head.block == nil {
```

Comment:
```suggestion
	if s.head == nil || s.head.block == nil || s.head.block.Block() == nil {
```

Should you check that block.Block() is not nil? Could panic on line 237 if it is nil.

Reply:
This is good. Thank you!

Reply:
Спасибо